### ISSUE TITLE
Add possibility to exclude views

### ISSUE THREAD

**Issue by SanderBenschop** [42BV/database-truncator#3]
Currently we are running into the issue that the H2 truncator attempts to truncate our views as well, which is of course not possible. It would be nice if it would either detect that an entity corresponds to a view and skip it or if the tool allows you to exclude table names based on patterns (like a 'view' postfix).

**Comment by robert-bor** [comment]
```java
database-truncator:
  exclude:
    - organisatie
    - "*view"
```

AntPathMatcher is used to match exclude values on.

**Comment by robert-bor** [comment]
Available in v0.3.0
